pypopquiz/io.py

```diff
@@ -112,6 +114,7 @@ def get_video(video_data: Dict[str, str], output_dir: Path) -> None:
         video = YouTube("https://www.youtube.com/watch?v={:s}".format(video_id))
         video = video.streams.filter(subtype=video_data["format"]).first()
         video.download(output_path=str(output_dir), filename=video_id)
-
+    elif video_source == "local":
+        shutil.copyfile(str(input_dir / get_video_file_name(video_data)), str(output_file))
```

Comment:
Since we use `pathlib`, can't we do something like this without using `shutil`:
```
input_file = input_dir / get_video_file_name(video_data)
input_file.rename(output_file)
```

Reply:
Forgot it could do that, will edit.